.gitignore

```diff
@@ -53,10 +53,14 @@ jsconfig.json
 # Built files
 packages/coinstac-ui/app/render/build
 
+packages/coinstac-api-server/rethinkdb_data
```

Comment:
What's up with this?

Reply:
RethinkDB's data directory. Just didn't want to sync it to Git as I can foresee problems doing that down the road.

Reply:
It's created automatically in whichever directory `rethinkdb` is run and I had just been running it in the api-server directory as that is where I start the server that uses rethink. We can put a flag on it to start it in a specific directory. That's fine.